web/src/lib/utils/helpers.tsx

```diff
@@ -377,3 +379,13 @@ export const flattenObject = (obj: any) => {
     return obj;
   }
 };
+
+export const getVisibleCertifications = (certifications: Certification[], userData: UserData) => {
+  if (!certifications) return [];
+  return certifications.filter((it) => !userData?.preferences.hiddenCertificationIds.includes(it.id));
+};
+
+export const getVisibleFundings = (fundings: Funding[], userData: UserData) => {
+  if (!fundings) return [];
+  return fundings.filter((it) => !userData?.preferences.hiddenFundingIds.includes(it.id));
+};
```

Comment:
maybe these could be in their own file? similar to how the filter functions are.